Tribler/Core/Config/tribler_config.py

```diff
@@ -309,16 +309,23 @@ def set_anon_proxy_settings(self, proxy_type, server=None, auth=None):
         self.config['libtorrent']['anon_proxy_type'] = proxy_type
         if server and proxy_type:
             self.config['libtorrent']['anon_proxy_server_ip'] = server[0]
-            self.config['libtorrent']['anon_proxy_server_ports'] = server[1]
+            # Convert the integers into strings for the config
+            self.config['libtorrent']['anon_proxy_server_ports'] = [str(i) for i in server[1]]
```

Comment:
It is not possible to save integers, it is not possible to save a list of integers.